julia/oxygen/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+Oxygen = "df9a0d86-3283-4920-82dc-4555fc0d1d8b"
```

Comment:
Do we need to keep hashes in this file ?
I mean computing the right version to use won't be done at startup by Julia ?

Reply:
We do need the UUID's - In Julia this is used to find the right package in the general registry. It's not tied to a specific version, that's what the `compat` section is for. If the `compat` section is dropped, then it will pull the latest version automatically

Reply:
Done! [5fae3de](https://github.com/the-benchmarker/web-frameworks/pull/7286/commits/5fae3de3237836c57e57c9822b1b1e3b59d46993)